front/package.json

```diff
@@ -21,6 +21,7 @@
     "framer-motion": "^10.12.17",
     "graphql": "^16.6.0",
     "hex-rgb": "^5.0.0",
+    "immer": "^10.0.2",
```

Comment:
interesting, we will add a section in docs discussing when it should be used or not